PiwikTracker/PiwikTracker.swift

```diff
@@ -96,6 +96,7 @@ final public class PiwikTracker: NSObject {
             guard events.count > 0 else {
                 // there are no more events queued, finish dispatching
                 self.isDispatching = false
```

Comment:
Just noticed that `isDispatching` is not thread safe, depending on what thread the caller calls the `dispatch()` method. There is a risk that the timer is not started/stopped as it should.